quasar.conf.js

```diff
@@ -138,6 +171,29 @@ module.exports = configure(function (ctx) {
           exclude: /node_modules/,
         })
 
+        // FIXME: delete, if not using the chainWebpack way
```

Comment:
I guess you meant to remove this bit :)

Reply:
It's noted in the description to delete this after I got your ok for the chain way :)
Deleted now

Reply:
Ah sorry, I was too brief in reading! All looks good though, thanks!